components/brave_rewards/common/features.h

```diff
@@ -23,6 +23,8 @@ BASE_DECLARE_FEATURE(kGeminiFeature);
 
 BASE_DECLARE_FEATURE(kVerboseLoggingFeature);
 
+BASE_DECLARE_FEATURE(kUnsupportedCustodialOptionsFeature);
```

Comment:
I'm trying to think about how to make this feature more self-documenting and less cryptic. Maybe one of these?

* `kAllowUnsupportedCustodiansFeature`
* `kAllowUnsupportedWalletProvidersFeature`.

Reply:
Yeah, both make more sense. I think we can go with `kAllowUnsupportedWalletProvidersFeature`.